schema-engine/connectors/sql-schema-connector/src/flavour/postgres.rs

```diff
@@ -657,6 +663,7 @@ pub(crate) enum Circumstances {
     IsCockroachDb,
     CockroachWithPostgresNativeTypes, // FIXME: we should really break and remove this
```

Comment:
hmm, unrelated to this pr but: we did break this case in Prisma 5 already, we can probably just remove it.